include/d/actor/d_a_sail.h

```diff
@@ -42,17 +45,32 @@ class daSail_packet_c : public J3DPacket {
     /* 0x1C3A */ u8 m1C3A;
     /* 0x1C3B */ u8 m1C3B;
     /* 0x1C3C */ J3DModel* m1C3C;
-    /* 0x1C40 */ u8 m1C40[0x1C44 - 0x1C40];
+    /* 0x1C40 */ s32 m1C40;
     /* 0x1C44 */ f32 m1C44;
     /* 0x1C48 */ f32 m1C48;
+    /* 0x1C4C */ f32 m1C4C;
+    /* 0x1C50 */ s16 m1C50;
+    /* 0x1C52 */ s16 m1C52;
+
+    inline Mtx *get_m0010() { return &this->m0010; }
+    inline Mtx *get_m0070() { return &this->m0070; }
+
+    inline cXyz* get_pos(s32 idx) {
+        return this->m00A4 + 0x54 * this->m1C3A + 0x54 * idx;
+    }
+    inline cXyz* get_nrm1(s32 idx) {
+        return this->m0C74 + 0x54 * this->m1C3A + 0x54 * idx;
+    }
+    inline cXyz* get_nrm2(s32 idx) {
+        return this->m1454 + 0x54 * this->m1C3A + 0x54 * idx;
+    }
 };
 
 class sail_class : public fopAc_ac_c {
 public:
     /* 0x0290 */ request_of_phase_process_class mClothPhase;
     /* 0x0298 */ request_of_phase_process_class mKaizokusenPhase;
     /* 0x02A0 */ daSail_packet_c mSailPacket;
-    /* 0x1EEC */ u8 m1EEC[0x1EF4 - 0x1EEC];
```

Comment:
this placeholder field was added to pad the size of `sail_class` to the correct size in its actor profile. removing this will make the size be incorrect.